diff --git a/src/libscotch/bgraph_bipart_df.c b/src/libscotch/bgraph_bipart_df.c
index 1c43bcc..a793f04 100644
--- a/src/libscotch/bgraph_bipart_df.c
+++ b/src/libscotch/bgraph_bipart_df.c
@@ -290,6 +290,9 @@ const BgraphBipartDfParam * const paraptr)        /*+ Method parameters +*/
   grafptr->commgainextn = commgainextn;
   grafptr->bbalval      = (double) ((grafptr->compload0dlt < 0) ? (- grafptr->compload0dlt) : grafptr->compload0dlt) / (double) grafptr->compload0avg;
 
+  // This plugs a memory leak.
+  memFree(loopdat.difotax + grafptr->s.baseval);
+
 #ifdef SCOTCH_DEBUG_BGRAPH2
   if (bgraphCheck (grafptr) != 0) {
     errorPrint ("bgraphBipartDf: inconsistent graph data");
