diff --git a/3rdparty/silo/src/silo/Makefile.am b/3rdparty/silo/src/silo/Makefile.am
index f7dd3bc..4b2dd24 100644
--- a/3rdparty/silo/src/silo/Makefile.am
+++ b/3rdparty/silo/src/silo/Makefile.am
@@ -79,9 +79,6 @@ EXTRA_DIST = \
  silo.h.in
 
 include_HEADERS = \
- silo.inc \
- silo_f77.inc \
- silo_f9x.inc \
  pmpio.h \
  silo_exports.h
 
@@ -105,16 +102,5 @@ FILES = \
  silo_ns.c \
  $(EXTRAFILES)
 
-BUILT_SOURCES = silo.inc silo_f9x.inc
 DISTCLEANFILES = silo_version.h
 
-# built if it doesn't exit, could cause problem with VPATH
-# be extra careful about check-in time-stamps! silo.inc MUST be the oldest.
-silo.inc: silo.h.in silo_f.h
-	cat ./silo.h $(srcdir)/silo_f.h | $(top_srcdir)/config/mkinc > $@
-
-silo_f77.inc: silo.inc
-	@ln -s $< $@ 2>/dev/null
-
-silo_f9x.inc: silo.h.in silo_f.h
-	cat ./silo.h $(srcdir)/silo_f.h | $(top_srcdir)/config/mkinc --f90 > $@
diff --git a/3rdparty/silo/src/silo/Makefile.in b/3rdparty/silo/src/silo/Makefile.in
index d785a95..b7db04f 100644
--- a/3rdparty/silo/src/silo/Makefile.in
+++ b/3rdparty/silo/src/silo/Makefile.in
@@ -139,8 +139,7 @@ LINK = $(LIBTOOL) --tag=CC $(AM_LIBTOOLFLAGS) $(LIBTOOLFLAGS) \
 	$(LDFLAGS) -o $@
 SOURCES = $(libsilo_silo_la_SOURCES)
 DIST_SOURCES = $(am__libsilo_silo_la_SOURCES_DIST)
-am__include_HEADERS_DIST = silo.inc silo_f77.inc silo_f9x.inc pmpio.h \
-	silo_exports.h silo_json.h
+am__include_HEADERS_DIST = pmpio.h silo_exports.h silo_json.h
 am__vpath_adj_setup = srcdirstrip=`echo "$(srcdir)" | sed 's|.|.|g'`;
 am__vpath_adj = case $$p in \
     $(srcdir)/*) f=`echo "$$p" | sed "s|^$$srcdirstrip/||"`;; \
@@ -354,8 +353,7 @@ EXTRA_DIST = \
  silo_json.c \
  silo.h.in
 
-include_HEADERS = silo.inc silo_f77.inc silo_f9x.inc pmpio.h \
-	silo_exports.h $(am__append_5)
+include_HEADERS = pmpio.h silo_exports.h $(am__append_5)
 nodist_include_HEADERS = \
  silo.h
 
@@ -372,7 +370,6 @@ FILES = \
  silo_ns.c \
  $(EXTRAFILES)
 
-BUILT_SOURCES = silo.inc silo_f9x.inc
 DISTCLEANFILES = silo_version.h
 all: $(BUILT_SOURCES)
 	$(MAKE) $(AM_MAKEFLAGS) all-am
@@ -710,17 +707,6 @@ uninstall-am: uninstall-includeHEADERS uninstall-nodist_includeHEADERS
 	uninstall-includeHEADERS uninstall-nodist_includeHEADERS
 
 
-# built if it doesn't exit, could cause problem with VPATH
-# be extra careful about check-in time-stamps! silo.inc MUST be the oldest.
-silo.inc: silo.h.in silo_f.h
-	cat ./silo.h $(srcdir)/silo_f.h | $(top_srcdir)/config/mkinc > $@
-
-silo_f77.inc: silo.inc
-	@ln -s $< $@ 2>/dev/null
-
-silo_f9x.inc: silo.h.in silo_f.h
-	cat ./silo.h $(srcdir)/silo_f.h | $(top_srcdir)/config/mkinc --f90 > $@
-
 # Tell versions [3.59,3.63) of GNU make to not export all variables.
 # Otherwise a system limit (for SysV at least) may be exceeded.
 .NOEXPORT:
